src/webapp/pages/event-tracker/EventTrackerPage.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { IconEdit24 } from "@dhis2/ui";
+
+import i18n from "../../../utils/i18n";
+import { Layout } from "../../components/layout/Layout";
+import { Section } from "../../components/section/Section";
+import { Button } from "../../components/button/Button";
+import { RiskAssessmentSection } from "./RiskAssessmentSection";
+
+export const EventTrackerPage: React.FC = React.memo(() => {
+    return (
+        <Layout title={i18n.t("Event Tracker")}>
+            <Section
+                title={"Event details, event date"}
+                titleVariant="secondary"
+                headerButtom={
+                    <Button
+                        onClick={function (): void {
+                            throw new Error("Function not implemented.");
+                        }}
```

Comment:
Move function to the module level.

Reply:
Also this was an example but I'm going to remove it